### ISSUE TITLE
Beorn's alternate form tile graphic not working

### ISSUE THREAD

**Issue by RogueFanUK** [angband/angband#4848]
When Beorn transforms into his bear form, the tile graphic isn't working, you see a lower case purple 'q' on a black background.

**Comment by RogueFanUK** [comment]
Yes, I'm using the Gervais set. I can confirm the bear form correctly displays in the Light and Dark versions of Shockbolt.

**Comment by RogueFanUK** [comment]
One thing of interest, the game defaults to the Gervais set?

**Comment by NickMcConnell** [comment]
Wow, really?  What OS are you playing on, and what version?

**Comment by RogueFanUK** [comment]
Windows 10 Home. I'm sure it defaults in 4.2.1 and 4.2.2 to Gervais.

**Comment by NickMcConnell** [comment]
I just downloaded both (from Github) for windows 10 and they defaulted to Shockbolt dark.  I'll try and see if there's anything weird that could be causing what you're seeing.

**Comment by NickMcConnell** [comment]
OK, I have it - Shockbolt tiles are used for larger screens, Gervais for smaller.

**Comment by RogueFanUK** [comment]
Ah - that would explain it. I'm on a relatively small screen laptop - 1920 x 1080.
